sqldelight-gradle-plugin/src/main/kotlin/com/squareup/sqldelight/gradle/GenerateMigrationOutputTask.kt

```diff
@@ -42,6 +42,9 @@ abstract class GenerateMigrationOutputTask : SourceTask(), SqlDelightWorkerTask
 
   @Input override var useClassLoaderIsolation = true
```

Comment:
I couldn't figure out a good way to hide this and make it a requirement of the interface

Reply:
I was also playing around with using task properties and hiding it that way, but I couldn't get it to work.

Reply:
abstract class instead of interface?

Reply:
Hm yeah that can work. I didn't realize they all inherited from SourceTask already.